README.md

```diff
@@ -98,29 +95,52 @@ execute `:runIde` Gradle task (`Gradle panel` -> `Tasks` -> `intellij` -> `runId
 
 To watch the logs of this IntelliJ instance, run `tail -f build/idea-sandbox/system/log/idea.log`.
 
+### Generate plugin zip
+
+To generate a plugin archive run `:buildPlugin` Gradle task (`Gradle panel` -> `Tasks` -> `intellij` -> `buildPlugin` or `./gradlew buildPlugin`).<br/>
+The resulting file will be available under `build/distributions/`.
 
-### Run UI tests
 
-```
-./scripts/run-ui-tests [<intellij-version>]
-```
+## Installing
 
-See [Gradle Intellij plugin docs](https://github.com/JetBrains/gradle-intellij-plugin/tree/master/examples/ui-test-example)
-for more details.
+### Install from JetBrains Marketplace
 
+This plugin is available on [JetBrains Marketplace](https://plugins.jetbrains.com/plugin/14221-git-machete). <br/>
+To install this plugin go to `File` -> `Settings` -> `Plugins`, then make sure you are on `Marketplace` tab (not `Installed`), in search box type `Git Machete` and click install.
+Plugin will be installed. After installation click `Restart IDE` and confirm that action in a messagebox.
 
-### Generate plugin zip
+### Install plugin zip
 
-To generate a plugin archive run `:buildPlugin` Gradle task (`Gradle panel` -> `Tasks` -> `intellij` -> `buildPlugin` or `./gradlew buildPlugin`).<br/>
-The resulting file will be available under `build/distributions/`.
+Go to `File` -> `Settings` -> `Plugins` -> `(gear icon)` -> `Install Plugin from Disk...`, select the zip and restart IDE.
 
 
-### Install snapshot build of the plugin from CI
+## Logging
```

Comment:
This part can remain in README... but more as a guideline on how to provide logs when reporting an issue; also, while you're here, consider adding ISSUE_TEMPLATE.md (or however the issue template file is now called in GitHub)

Reply:
About issue template: now it looks like this https://docs.github.com/en/github/building-a-strong-community/configuring-issue-templates-for-your-repository